source/js/buyers-guide/components/creepometer/creepometer.jsx

```diff
@@ -6,7 +6,7 @@ export default class Creepometer extends React.Component {
 
     this.faceCount = 40; // Number of face frames
     this.faceHeight = 70; // pixel height for one frame
-    this.framePath = `/_images/buyers-guide/faces/`;
+    this.framePath = `../../../../static/_images/buyers-guide/faces/`;
```

Comment:
@Pomax @mmmavis @cadecairos Any idea on how to make that less ugly? The url for that file is `root_url/static/_images/...`. If I don't do that, it resolves to `root_url/products/NAME/_images/...` which of course does a 404.